Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/SwiftUIScreenDemoViewController.swift

```diff
@@ -0,0 +1,176 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import HorizonCalendar
+import SwiftUI
+
+// MARK: - SwiftUIScreenDemoViewController
+
+final class SwiftUIScreenDemoViewController: UIViewController, DemoViewController {
+
+  // MARK: Lifecycle
+
+  init(monthsLayout: MonthsLayout) {
+    self.monthsLayout = monthsLayout
+    super.init(nibName: nil, bundle: nil)
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Internal
+
+  let calendar = Calendar.current
+  let monthsLayout: MonthsLayout
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    title = "SwiftUI Screen"
+
+    let hostingController = UIHostingController(
+      rootView: SwiftUIScreenDemo(calendar: calendar, monthsLayout: monthsLayout))
+    addChild(hostingController)
+
+    view.addSubview(hostingController.view)
+    hostingController.view.translatesAutoresizingMaskIntoConstraints = false
+    NSLayoutConstraint.activate([
+      hostingController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+      hostingController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+      hostingController.view.topAnchor.constraint(equalTo: view.topAnchor),
+      hostingController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+    ])
+
+    hostingController.didMove(toParent: self)
+  }
+
+}
+
+// MARK: - SwiftUIScreenDemo
+
+struct SwiftUIScreenDemo: View {
+
+  // MARK: Lifecycle
+
+  init(calendar: Calendar, monthsLayout: MonthsLayout) {
+    self.calendar = calendar
+    self.monthsLayout = monthsLayout
+
+    let startDate = calendar.date(from: DateComponents(year: 2023, month: 01, day: 01))!
+    let endDate = calendar.date(from: DateComponents(year: 2026, month: 12, day: 31))!
+    visibleDateRange = startDate...endDate
+
+    monthDateFormatter = DateFormatter()
+    monthDateFormatter.calendar = calendar
+    monthDateFormatter.locale = calendar.locale
+    monthDateFormatter.dateFormat = DateFormatter.dateFormat(
+      fromTemplate: "MMMM yyyy",
+      options: 0,
+      locale: calendar.locale ?? Locale.current)
+  }
+
+  // MARK: Internal
+
+  var body: some View {
+    CalendarViewRepresentable(
+      calendar: calendar,
+      visibleDateRange: visibleDateRange,
+      monthsLayout: monthsLayout,
+      dataDependency: calendarSelection)
+
+    .verticalDayMargin(8)
+    .horizontalDayMargin(8)
+    .interMonthSpacing(16)
+
+    .monthHeaderItemProvider { month in
+      let monthHeaderText = monthDateFormatter.string(from: calendar.date(from: month.components)!)
+      if case .vertical = monthsLayout {
+        return HStack {
+          Text(monthHeaderText).font(.title2)
```

Comment:
```suggestion
          Text(monthHeaderText)
          .font(.title2)
```

move these to the next line for consistency?